### DISCUSSION TITLE
You are forbidden from performing this action on github.com

### DISCUSSION THREAD

**Discussion by connectfoundation** [community/community#36852] [GitHub Education]
hello.
We are operating a classroom with about 200 people, but the following error occurs to students when they join group assignments.

"You are forbidden from performing this action on github.com"
![ssss](https://user-images.githubusercontent.com/50396533/197094614-9ca96b53-d02b-4045-bea8-bb27a81bdcb0.jpg)

How should we respond?

thank you

**Comment by ossanna16** [comment]
Thank you for asking a good question @connectfoundation, I'm glad you found our community 🙂
